src/tox/package/__init__.py

```diff
@@ -0,0 +1,68 @@
+import py
```

Comment:
personally I dislike code in `__init__.py` -- hard to find, easy to make circular imports among other things.  not blocking though

Reply:
> personally I dislike code in __init__.py -- hard to find, easy to make circular imports among other things. not blocking though

👍, for the same reasons hehehe

Reply:
I'm not sure in this case. Would lead to some common.py/base.py files instead. Or package.py inside package module?